### ISSUE TITLE
info/error are not defined

### ISSUE THREAD

**Issue by smazurov** [grampajoe/step-opsworks-deploy#17]
when running the step, I get warnings like 

```
/---snip----/
/wercker/steps/grampajoe/opsworks-deploy/1.0.1/install.sh: 3: /wercker/steps/grampajoe/opsworks-deploy/1.0.1/install.sh: info: not found
/---snip----/
pajoe/opsworks-deploy/1.0.1/deploy.sh: 51: /wercker/steps/grampajoe/opsworks-deploy/1.0.1/deploy.sh: info: not found
/---snip----/
```
my guess is because `info` (or `error`) isn't a thing. 

looking at tests, you're mocking those (which isn't super useful since the output is never checked), but i don't see them defined for real anywhere. 

Additionally, running apt-get on `nodejs` adds some needless time and output to the whole process, why do you need to upgrade node from 0.10.28 to 0.10.33?

All in all, awesome step, saves me a ton of time.  Love the `wait-for-deploy` too.

**Comment by grampajoe** [comment]
Hi @smazurov, thanks for opening this issue. I'm glad this step is helpful!

According to the [step docs](http://devcenter.wercker.com/articles/steps/guide.html#toc_5), the `info` and `error` functions should be defined. They're defined in a script that supposedly gets run before each step: https://github.com/wercker/wercker-init/blob/master/init.sh

It might be that `install.sh` and `deploy.sh` don't pick up the functions defined in the parent environment, but I don't know enough about how all that works to say for sure. Testing these things is difficult, because you have to publish a release to run it in the real Wercker environment. That would be why I have all those functions mocked out in test helper scripts, and why the changelog is full of bugfix releases.

As for the node install, I'm just not assuming node will already be present on whichever box is used. It could definitely check first, though, before doing that extra work.

I'll try to address these things and get them released soon. Please don't hesitate to bring up any other issues you find, and thanks again for the feedback!

**Comment by smazurov** [comment]
Yeah, the scripts are run with `source` so those commands are only available in `run.sh`. 
```bash
$ source '/wercker/wercker-build-essentials/init.sh'
$ source "$WERCKER_STEP_ROOT/run.sh"
```

regarding node, its part of their [webessentials](https://github.com/wercker/box-ubuntu12.04-webessentials) box which I believe you have to inherit from (could be wrong), but its certainly not a bad idea to check anyways.

**Comment by grampajoe** [comment]
Ok, so maybe it should be `source` all the way down. Thanks for looking this up!

**Comment by grampajoe** [comment]
Hi again, @smazurov. These issues should (finally!) be addressed in version 1.0.2 of the step.
